packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -41,6 +41,41 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
   @override
   ContainerLayer get parent => super.parent;
 
+  // Whether this layer has any changes since it was last sent to the engine
+  // using [addToScene]. Initialized to true as a new layer has never called
+  // [addToScene].
+  bool _isDirty = true;
+
+  /// Mark that this layer has changed and we need to upload changes to the
+  /// engine using [addToScene].
+  void markDirty() { _isDirty = true; }
```

Comment:
newline after `{`

Reply:
Changed to `void markDirty() => _isDirty = true;`. Sorry that this PR is broken again (I can't view the diff). I'm creating a new PR with every commit squashed. Hopefully it won't break.